packages/console/src/pages/ConnectorDetails/components/ConnectorContent.tsx

```diff
@@ -42,9 +44,26 @@ const ConnectorContent = ({ isDeleted, connectorData, onConnectorUpdated }: Prop
   } = methods;
 
   useEffect(() => {
-    const { name, logo, logoDark, target } = connectorData.metadata;
+    const { formItems, metadata } = connectorData;
+    const { name, logo, logoDark, target } = metadata;
     const { config, syncProfile } = connectorData;
```

Comment:
nit: can refactor these lines in a single destructure operation to make it neat

Reply:
updated